### DISCUSSION TITLE
Curation report - October 24 - 30

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/GPML-Repo#97] [Curation reports]
Hi all,
here is my curation report: 
Not too many changes this time (26 pathways)
@DeniseSl22: is https://www.wikipathways.org/index.php/Pathway:WP5155 interesting for the lipid maps portal? It looks quite extensive, but is not approved at the moment. For me, it is difficult to assess if the information is already included in the other eicosanoid metabolism pathways (which are homology converted).

@fehrhart: https://www.wikipathways.org/index.php/Pathway:WP5298 seems to be an addition to https://www.wikipathways.org/index.php/Pathway:WP5200, but in mouse. Is this interesting for the rare disease portal? Currently it is still "under construction", but it looks quite complete already. The author did some changes to the original pathway as well, adding excitatory pyramidal neurons.

https://www.wikipathways.org/index.php/Pathway:WP5296 and https://www.wikipathways.org/index.php/Pathway:WP5297 seem to be duplicates, I added a discussion on the second one to ask if this is on purpose or should be deleted.

https://www.wikipathways.org/index.php/Pathway:WP5294 looks quite complete. The only issue I see is that some proteins as AKT and NOS are just labels, and the pathway includes a label legend stating which proteins they could be. Do we have a tutorial page explaining how to treat such groups of proteins? I could not find one, and I must admit I'm myself always confusing groups and complexes.
@DeniseSl22: you did some work on https://www.wikipathways.org/index.php/Pathway:WP5280. What would you say is still missing to remove the needs work tag? There is no description with it.

@egonw: https://www.wikipathways.org/index.php/Pathway:WP5077 was tagged as for deletion (due to missing database identifiers according to the discussion page). Do you agree with deleting it?

I removed under construction and added needs work for https://www.wikipathways.org/index.php/Pathway:WP5242

A general question regarding deletions: There are two pathways https://www.wikipathways.org/index.php/Pathway:WP5259 and https://www.wikipathways.org/index.php/Pathway:WP5263 which were probably started at the same workshop. As they are not the usual tutorial pathways, I thought the usual process would be to add the "incomplete" tag, rather than request deletion already. Or did this process change as we are switching to the new platform soon?

Same for https://www.wikipathways.org/index.php/Pathway:WP5236, which doesn't look like a tutorial pathway at all. @fehrhart, do you agree with deleting this pathway?

https://www.wikipathways.org/index.php/Pathway:WP1423 is marked with both approved and incomplete, as it could be potentially merged with https://www.wikipathways.org/index.php/Pathway:WP4142. Is someone familiar with sphingolipid metabolism to assess this properly?

I deleted https://www.wikipathways.org/index.php/Pathway:WP4277, and identified an identical pathway which is probably also a tutorial pathway. I tagged it for deletion as well: https://www.wikipathways.org/index.php/Pathway:WP5210

**Comment by AlexanderPico** [comment]
Wow. Lots of good stuff in this report! Here are some things I did and remaining questions are in bold:

1. https://www.wikipathways.org/index.php/Pathway:WP5155 - marked as approved. **Add to lipid map community?**
2. https://www.wikipathways.org/index.php/Pathway:WP5298 - marked as approved and featured. **Add this mouse version to rare disease community?**
3. https://www.wikipathways.org/index.php/Pathway:WP5296 - I know the author and emailed them directly in case WP notifications don't work.
4. https://www.wikipathways.org/index.php/Pathway:WP5294 - marked as approved.
5. https://www.wikipathways.org/index.php/Pathway:WP5280 - **@DeniseSl22 ready to mark as approved?**
6. https://www.wikipathways.org/index.php/Pathway:WP5077 - removed delete tag.
7. Re: novel tutorial pathways. If abandoned (>30 days), then they should be marked for deletion (as they were). I have deleted them now. Indeed, we will not have "stub" or "incomplete" pathways in the new system.
8. https://www.wikipathways.org/index.php/Pathway:WP5236 - essentially complete. I removed delete tag, added a few annotations and marked as approved.
9. https://www.wikipathways.org/index.php/Pathway:WP1423 - fixed interaction and removed incomplete tag.

**Comment by fehrhart** [comment]
*Same for https://www.wikipathways.org/index.php/Pathway:WP5236
<https://www.wikipathways.org/index.php/Pathway:WP5236>, which doesn't look
like a tutorial pathway at all. @fehrhart <https://github.com/fehrhart>, do
you agree with deleting this pathway? *

Nope, please don't delete - that is a pathway that describes the current
knowledge about kidney development including the biomarkers that mark the
different time and spatial developing cells and tissues. There is also a
paper in preparation about this (and some other CAKUT) pathways.

Best regards,
Freddie

**Reply by danidi** [reply]
I thought so, that's why I brought it up. Will authors of a pathway receive an information if the pathway is marked for deletion?

**Comment by Chris-Evelo** [comment]
1.  https://www.wikipathways.org/index.php/Pathway:WP5298 - marked as approved and featured. Add this mouse version to rare disease community?
In principle I would say yes. Although we might want to look for a way to highlight the fact that this is a non-human pathway even more. If there is a mouse model that can be used to study a human rare disease we might indeed want to add that. But in this specific case I am confused a bit by the human and mouse pathways,
I checked the human version of the Dravet pathway from what I understand they are both about the same gene being mutated, although the mouse pathway talks both about point mutations and truncating mutations (or is that meant to be a point mutation leading to translation truncation?). I am not sure about a number of things:

  1.  Why are different arrow types being used?
  2.  What is the meaning of the right hand side?
  3.  Nav1.1 seems to be phosphorylated? Is it then not useful to add by what?
  4.  The mouse pathway has a lot more information, assuming that is correct and relevant should that not also be added to the human version? (Should we maybe update one to be complete and create the other using homology mapping?)
  5.  Is everything relevant, like are glucose and glycolysis really needed in this pathway? If so we might want to describe why in the description
  6.  The description mentions a core gene (and the mouse one a few other culprits for disease initiation. Should we somehow highlight=t such genes?
Best, Chris



From: Alexander Pico ***@***.***>
Reply to: wikipathways/wikipathways-development ***@***.***>
Date: Sunday, 30 October 2022 at 21:06
To: wikipathways/wikipathways-development ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [wikipathways/wikipathways-development] Curation report - October 24 - 30 (Discussion #97)


Wow. Lots of good stuff in this report! Here are some thing I did and remaining questions in bold:

  1.  https://www.wikipathways.org/index.php/Pathway:WP5155 - marked as approved. Add to lipid map community?
  2.  https://www.wikipathways.org/index.php/Pathway:WP5298 - marked as approved and featured. Add this mouse version to rare disease community?
  3.  https://www.wikipathways.org/index.php/Pathway:WP5296 - I know the author and emailed them directly in case WP notifications don't work.
  4.  https://www.wikipathways.org/index.php/Pathway:WP5294 - marked as approved.
  5.  https://www.wikipathways.org/index.php/Pathway:WP5280 - @DeniseSl22<https://github.com/DeniseSl22> ready to mark as approved?
  6.  https://www.wikipathways.org/index.php/Pathway:WP5077 - removed delete tag.
  7.  Re: novel tutorial pathways. If abandoned (>30 days), then they should be marked for deletion (as they were). I have deleted them now. Indeed, we will not have "stub" or "incomplete" pathways in the new system.
  8.  https://www.wikipathways.org/index.php/Pathway:WP5236 - essentially complete. I removed delete tag, added a few annotations and marked as approved.
  9.  https://www.wikipathways.org/index.php/Pathway:WP1423 - fixed interaction and removed incomplete tag.

**Reply by danidi** [reply]
Thank you, Chris, these are interesting points. Does someone know the author, so we could forward the discussion?

**Comment by fehrhart** [comment]
I am working with that group from LMU (Munich University) on these pathways
- will take care about that with them!

**Comment by DeniseSl22** [comment]
Thanks for the overview @danidi ! To comment on the points you addressed to me:

- [WP5155](https://www.wikipathways.org/index.php/Pathway:WP5155) I've contacted the main author, to discuss the reason for creating this PWM (since we already have some of them; the homology converted ones have been checked manually, the originals are from LIPID MAPS, mouse species). From the author: "The pathways I created are intended to used as part of a data visualization pipeline; however, I have not completed it yet. I might be interested in merging pathways now or in the future, but I will need more information to ensure compatibility with my project. " I'll follow up on this after I've finished my thesis ;) 
- [WP5280](https://www.wikipathways.org/index.php/Pathway:WP5280) This is part of a collaboration with the LIPID MAPS team (as is WP5279+WP5277; These Pws are still under construction, I've added that label for clarification.

**Comment by fehrhart** [comment]
Hi all,

 not much to report, a lot of small changes in titles and descriptions by
Eric.

One thing... I could for the first time not do the curation via
WikiPathways Academy using Firefox browser, I had to open Chrome for that.
The pathways would not load, see screenshot below.

Best regards,
Freddie

[image: grafik.png]
